primes.rb

```diff
@@ -1,6 +1,21 @@
 class Primes
   def self.sum_to(limit = 100)
     # TODO - add your prime number solution here...
-    "I'm working on it!"
-  end
-end
\ No newline at end of file
+    primes = (2..limit).select do |n|
+    	(2..Math.sqrt(n)).none? do |i|
+    		(n%i).zero?
+    	end
+    end
+
+    	sum = primes.inject { |sum, i| sum + i }
+	end
+end
+
+Primes.sum_to(100)
+Primes.sum_to(1000)
+Primes.sum_to(10000)
+
+#puts Primes.sum_to(2000000)
+#commented out the 2 million soloution as it takes a while to run but it does work
```

Comment:
I went and did a little more research into it using stackoverflow, google and the documentation. I found i can use the prime method which ruby already has. When I tested this method... it worked much faster and also looks a lot more tidy. Here it is below:

```
require 'prime'

class Primes
  def self.sum_to(limit)
    Prime.take_while {|p| p < limit}.reduce(:+)
  end 
end

 puts Primes.sum_to(10000)
 puts Primes.sum_to(1000)
 puts Primes.sum_to(100)
 puts Primes.sum_to(2000000)
```

Please let me know if you like for me to push this new method into my profile.